File: Python/Length of Last Word.py
class Solution:
    # @param s, a string
    # @return an integer
    def lengthOfLastWord(self, s):
        if not s:
            return 0
        s = s.strip().split()
        print s
        if not s or len(s) == 0:
            return 0
        return len(s[-1])
        
s = Solution()
print s.lengthOfLastWord(" ")

File: Java/src/Length of Last Word.java
public class Solution {
    public int lengthOfLastWord(String s) {
        int length = s.length();
        int ret = 0;
        for (int i = length-1; i>=0 ; i--){
            if (ret == 0 && s.charAt(i) != ' '){
                ret = 1;
            }
            else if (ret != 0 && s.charAt(i) != ' '){
                ret++;
            }
            else if (ret != 0 && s.charAt(i) == ' '){
                return ret;
            }
        }
        return ret;
        
    }
}